### ISSUE TITLE
Cartopy now requires Proj version 8.0.0 but webpage says only needs 4.9.0

### ISSUE THREAD

**Issue by bismurphy** [SciTools/cartopy#1879]
This appears to be a result of the very recent latest release.

Attempting to install Cartopy on my machine says ` Proj version 6.3.1 is installed, but cartopy requires at least version 8.0.0.`

Contrarily, https://scitools.org.uk/cartopy/docs/latest/installing.html#installing indicates Proj only needs to be 4.9.0 or better. I'm struggling to get my Proj to version 8.0.0, but that's a separate issue from Cartopy.

Would it be possible to make the 8.0.0 Proj optional? It appears to only be used for one particular feature.

**Comment by dopplershift** [comment]
The reason the docs still say Proj 4.9 is because those docs are for 0.19. We are waiting on SciTools/scitools.org.uk#245 to get merged to get the 0.20 docs posted, which should be correct.

With regards to making 8.0 optional, I believe there were reasons in #1808 that we needed to move beyond Proj 6.3. @snowman2 can you add anything about that?

**Comment by snowman2** [comment]
@dopplershift, the error comes from the `setup.py` which is only used for the geodesic stuff. There really isn't any reason that the pin should be 8.0.0 in the setup,py. All of the stuff from `pyproj` that requires PROJ 8+ could be installed via the pyproj wheels. Looks like the move in the pin happened in #1854.

**Comment by snowman2** [comment]
> There really isn't any reason that the pin should be 8.0.0 in the setup,py

Actually, I take that back. This setup has the potential to be unstable and isn't tested in the cartopy CI tests, so the pin at PROJ 8+ makes sense for now.

**Comment by MHBalsmeier** [comment]
This is, however, a bit problematic, since it makes `0.20.0` unusable on Debian distros if one does not want to install proj from source. ... And most people (me included) prefer `apt` or something similar and this does not gove `proj8`.

**Comment by snowman2** [comment]
Once #1832 is resolved, this shouldn't be an issue (assuming you don't mind using the `pyproj` wheels).

**Comment by MHBalsmeier** [comment]
> Once #1832 is resolved, this shouldn't be an issue (assuming you don't mind using the `pyproj` wheels).

I don't, but most people don't install by cloning the current repository but rather by installing the PyPi release with pip.

**Comment by tfardet** [comment]
Forgive me if this is not the right thread for that but, in line with previous questions, I was wondering whether it would be possible to make cartopy a little more backward compatible and extend support for proj < 8 in order to make life easier for LTS users that are still using 6 something?
Or did the previous message mean that cartopy will soon automatically install the right pyproj when installed from pip and that this would also work on old LTS because the .so file would be included in pyproj?

**Comment by riddella** [comment]
Supporting a slightly earlier version of libproj would be very welcome.

- Debian bullseye and Ubuntu 21.10 both ship libproj 7.2.1.

**Comment by RaviBeagle** [comment]
I see that the issue is still open yet. But is there any workaround possible ? Thanks

**Comment by greglucas** [comment]
v0.21 is out and there is no longer a minimum required PROJ. All PROJ requirements are handled by pyproj now.
